storageFormats/eventMetadata.cc

```diff
@@ -365,3 +365,45 @@ std::string rpwa::eventMetadata::getStringForEventsType(const eventsTypeEnum& ty
 	}
 	return "UNKNOWN";
 }
+
+
+bool
+additionalVariables::setBranchAddresses(const eventMetadata& metaData)
+{
+	TTree* tree = metaData.eventTree();
+	if (tree != nullptr){
+		for(const auto& label: metaData.additionalSavedVariableLables()){
+			_labels.push_back(label);
+			int err = tree->SetBranchAddress(label.c_str(), &_additionalSavedVariables[label]);
```

Comment:
It may help the reader to add parantheses
`&(_additionalSavedVariables[label])`